app/src/main/res/layout/quest_answer_fragment.xml

```diff
@@ -1,110 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <de.westnordost.streetcomplete.view.SlidingRelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
     <android.support.design.widget.CoordinatorLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_above="@+id/buttonPanel"
-        android:background="#00000000"
-        >
+        android:layout_marginStart="-9dp"
+        android:layout_marginEnd="-9dp"
+        android:layout_marginTop="-8dp"
+        android:layout_marginBottom="-8dp"
```

Comment:
Is it deliberate that you sometimes have -9dp and sometimes -8dp? (also in the `land`-version)

Reply:
I thought the elevation should only be displayed at the right side.

Reply:
https://github.com/westnordost/StreetComplete/pull/523#issuecomment-325635910